pkg/kritis/admission/admission.go

```diff
@@ -339,8 +368,8 @@ func getReviewer(client metadata.Fetcher) reviewer {
 	})
 }
 
-// reviewer interface defines an Kritis Reviewer Struct.
-// TODO: This will be removed in future refactoring.
+// reviewer interface defines Kritis Reviewer struct.
 type reviewer interface {
```

Comment:
Does this interface need to exist? Can the underlying type just be used instead?

Reply:
The interface is useful for mocking it in the admission_test. I added a comment and enhanced the tests for this.